packedForest/src/forestTypes/basicForests/rfClassification/rfTree.h

```diff
@@ -95,7 +99,15 @@ namespace fp{
 
 				inline void checkOOB(){
 					totalOOB += nodeQueue.back().returnOutSampleSize();
-					correctOOB += nodeQueue.back().returnOutSampleError(tree.back().returnClass());
+					correctOOB += nodeQueue.back().returnOutSampleCorrect(tree.back().returnClass());
+				}
+
+				inline int returnTotalOOB(){
+					return this->totalOOB;
```

Comment:
Shouldn't we be able to just `return totalOOB` instead of `this->totalOOB` just like `returnOOB()` returns OOBAccuracy?  

If not, @jbrowne6 do you know why?

Reply:
I changed it and didn't get any errors in my tests.